hicommonwealth/commonwealth
# [ERC20 Launchpad] - Endpoint integration for Launch token step

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/9207

## Description of Changes
Added endpoint integration for Launch token step

## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_TOKENIZED_COMMUNITY`
- Visit `/createTokenCommunity`
- Verify you see the auth modal popup with only EVM wallet options (if you are not logged in already)
- Verify you see the address selection modal popup (if you are logged in)
- If you close the auth modal or the address selection modal without authentication, then verify you get these modals when you try to submit.
- Fill the form - verify the form validation works as expected
- Submit - the form submission will fail with this error ATM (plz see "Other Considerations" for more info)
<img width="671" alt="image" src="https://github.com/user-attachments/assets/36b1b770-8e4f-4017-83a1-22db7b0f4038">

## Deployment Plan
N/A

## Other Considerations
- ATM the form submission will throw an error like this, it will be fixed when we deploy the token specific abis (cc: @ianrowan)
<img width="671" alt="image" src="https://github.com/user-attachments/assets/36b1b770-8e4f-4017-83a1-22db7b0f4038">

- There are some `TODO's` left intentionally in the code that relates to adding some token-specific information, these will be done in a followup.
